### PR TITLE
fix: outside container background color issue

### PR DESCRIPTION
**Pull request by jenisha00**

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -324,17 +323,12 @@ function colormag_content_width() {
...
 			}
-		} else {
-			if ( 'no_sidebar_full_width' === $colormag_default_sidebar_layout ) {
+		} elseif ( 'no_sidebar_full_width' === $colormag_default_sidebar_layout ) {
 				$content_width = 1140; /* pixels */
-			}
 		}
-	} else {
-		if ( 'no_sidebar_full_width' === $layout_meta ) {
+	} elseif ( 'no_sidebar_full_width' === $layout_meta ) {
```
Comment (asheshmagar) [inline functions.php:329]: Are these changes intended? @jenisha00

Comment (jenisha00) [reply functions.php:329]: No, these changes are not intended but occurred due to formatting. @asheshmagar

Comment (asheshmagar) [reply functions.php:329]: Then revert them formatting tries to convert the lonely if inside else to elseif but in some condition lonely if inside else if required
